src/model/data_access_layer/mustwatches/mustwatches_reader.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Tuple
+
+from sqlalchemy.orm.query import Query
+
+from src.model.tables import Mustwatches
+
+
+class MustwatchesReader:
+
+    def __init__(self, alchemist):
+        self.session = alchemist.session
+
+    def __get_list_from_raw_list(self,
+                                 raw_list: List[Tuple[int]]) -> List[int]:
+        return [raw_list[i][0] for i in range(len(raw_list))]
+
+    def get_watches_id_list_for_add_to_one_chosen_user(self,
+                                                       chosen_user_tuple: Tuple[int]) -> List[int]:
+        chosen_user_id = chosen_user_tuple[0]
+        raw_chosen_user_watches_id = self.session.query(Mustwatches.watches_id).filter(
+            Mustwatches.users_id == chosen_user_id
+        ).all()
+        return self.__get_list_from_raw_list(raw_chosen_user_watches_id)
+
+    def get_watches_id_list_for_add_to_one_user(self,
```

Comment:
You don't need to name method for it external purpose. It will be more clear if you name it by it internal logic - like what it exact do.